src/reformatters/contrib/noaa/ndvi_cdr/ndvi_cdr/analysis/template_config.py

```diff
@@ -0,0 +1,275 @@
+from collections.abc import Sequence
+from typing import Any
+
+import numpy as np
+import pandas as pd
+from pydantic import computed_field
+
+from reformatters.common.config_models import (
+    BaseInternalAttrs,
+    Coordinate,
+    CoordinateAttrs,  # noqa: F401
+    DatasetAttributes,
+    DataVar,
+    DataVarAttrs,  # noqa: F401
+    Encoding,  # noqa: F401
+    StatisticsApproximate,  # noqa: F401
+)
+from reformatters.common.template_config import (
+    SPATIAL_REF_COORDS,  # noqa: F401
+    TemplateConfig,
+)
+from reformatters.common.types import AppendDim, Dim, Timedelta, Timestamp
+from reformatters.common.zarr import (
+    BLOSC_2BYTE_ZSTD_LEVEL3_SHUFFLE,  # noqa: F401
+    BLOSC_4BYTE_ZSTD_LEVEL3_SHUFFLE,  # noqa: F401
+    BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE,  # noqa: F401
+)
+
+# Constants for QA data variable
+QA_NETCDF_VAR_NAME = "QA"
+QA_FILL_VALUE = -32767
+QA_ENCODING_FILL_VALUE = -32767
+
+
+class NoaaNdviCdrInternalAttrs(BaseInternalAttrs):
+    """
+    Variable specific attributes used internally to drive processing.
+    Not written to the dataset.
+    """
+
+    netcdf_var_name: str
+    fill_value: float
+    scale_factor: float | None = None
+    add_offset: float | None = None
+    valid_range: tuple[float, float] | None = None
+
+
+class NoaaNdviCdrDataVar(DataVar[NoaaNdviCdrInternalAttrs]):
+    attrs: DataVarAttrs
```

Comment:
```suggestion
    pass
```
attrs is already defined like this on the parent class